app/code/Magento/ConfigurableProduct/view/frontend/web/js/configurable.js

```diff
@@ -740,10 +740,11 @@ define([
          * @private
          */
         _displayTierPriceBlock: function (optionId) {
-            var options, tierPriceHtml;
+            var options, tierPriceHtml, tierPrices = this.options.spConfig.optionPrices[optionId].tierPrices;
 
             if (typeof optionId != 'undefined' &&
-                this.options.spConfig.optionPrices[optionId].tierPrices != [] // eslint-disable-line eqeqeq
+                tierPrices != [] && // eslint-disable-line eqeqeq
```

Comment:
Thank you for your contribution!

Could you please change this check while we're at it? Right now it doesn't make any sense. I believe that we are trying to check here if `tierPrices` is an array so `_.isArray` could be used instead. This will also allow us to disable this ESLint comment.

Reply:
sure, will update

Reply:
I have updated. Can you please review now?